conda_index/index/convert_cache.py

```diff
@@ -167,29 +166,6 @@ def basename(path):
         )
 
 
-def extract_cache(path):
```

Comment:
Converting archives of the old cache, only needed during development of new-conda-index.